docs/en/contributor/Developer-Tools.md

```diff
@@ -9,8 +9,6 @@ priority: 3
 * Table of Contents
 {:toc}
 
-TODO(Andrew, Zac): Migrate content from Code-Conventions, add missing content
```

Comment:
is this addressed?

Reply:
Yep, Zac updated it a while back (https://github.com/Alluxio/alluxio/commit/0c21429bf3fbe123f475f2f49e5a7a456937b540#diff-ca749b9e00760e2485e32f19f61edec5) but forgot to remove the TODO

Reply:
ack